pkg/proc/native/threads_freebsd.go

```diff
@@ -125,3 +125,8 @@ func (t *nativeThread) ReadMemory(data []byte, addr uint64) (n int, err error) {
 func (t *nativeThread) withDebugRegisters(f func(*amd64util.DebugRegisters) error) error {
 	return proc.ErrHWBreakUnsupported
 }
+
+// SoftExc returns true if this thread received a software exception during the last resume.
+func (t *nativeThread) SoftExc() bool {
+	return false
```

Comment:
Why hard code this to false?

Reply:
The freebsd backend never kept track of which thread got a SIGTRAP, this is one more thing that will have to be done when it gets brought up to par with other backends. Always returning false is the safe thing to do for backends because handleHardcodedBreakpoint will always check trapthread.